src/system/Zikula/Module/SearchModule/AbstractSearchable.php

```diff
@@ -13,6 +13,7 @@
 
 namespace Zikula\Module\SearchModule;
 
+use Symfony\Component\DependencyInjection\ContainerInterface;
```

Comment:
it would mean refactoring current implementations in multiple modules as well as /system modules. I think Pages, Dizkus and maybe a few others (News??) also have the current implementation.